src/FlexExampleApp.mxml

```diff
@@ -1,14 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<s:ViewNavigatorApplication xmlns:fx="http://ns.adobe.com/mxml/2009" 
-							xmlns:s="library://ns.adobe.com/flex/spark" xmlns:mx="library://ns.adobe.com/flex/mx" firstView="views.FlexExampleAppHomeView" applicationDPI="160">
-	<fx:Declarations>
-		<!-- Place non-visual elements (e.g., services, value objects) here -->
-	</fx:Declarations>
+<s:ViewNavigatorApplication
+        xmlns:fx="http://ns.adobe.com/mxml/2009"
+        xmlns:s="library://ns.adobe.com/flex/spark"
+        width="100%" height="100%"
+
+        creationComplete="onCreationComplete()"
+        firstView="views.FlexExampleAppHomeView"
+        firstViewData="{appModel}"
+        >
+
 	<fx:Script>
 		<![CDATA[
-			import com.heyzap.sdk.ads.HeyzapAds;
-		]]>
+        import com.heyzap.sdk.ads.HeyzapAds;
+
+        import model.AppModel;
+
+        [Bindable]
+        private var appModel:AppModel = new AppModel();
+
+        protected function onCreationComplete():void {
+
+            HeyzapAds.getInstance().start("9317e0b9c4eaf41ed68e907e22ab3c5a", HeyzapAds.FLAG_NONE);
```

Comment:
The main idea of this test application to give developers ability to test how heyzap works.
So we have pre setup all package names and this app id. To get it just working without any additional steps for developers. Maybe we can add some other ID or just leave it empty? I tried with empty string and in some cases it didn't work well, so I put my ID there. What do you think?